setup.py

```diff
@@ -34,6 +34,30 @@ def get_path(fname):
     "Markdown>=2.6,<2.7",
 ]
 
+test_requirements = [
+    'django-functest>=1.0,<1.1',
+    'mock>=2.0,<2.1',
+    'pytest>=3.4,<3.5',
+    'pytest-django>=3.1,<3.2',
+    'pytest-cov>=2.4,<2.5',
+    'pytest-pythonpath>=0.7,<0.8',
+]
+
+test_lint_requirements = [
+    'flake8>=3.5,<3.6',
+]
+
+setup_requirements = [
+    'pytest-runner',
+]
+
+development_requirements = test_requirements + test_lint_requirements
+
+extras_requirements = {
+    'devel': development_requirements,
+    'test': test_requirements,
+    'testlint': test_lint_requirements,
+}
```

Comment:
I love this pattern!! Really nice overview of requirements, everything maintained in the same place, and the loosest form of dependencies that affect distribution and testing at the same time.